src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Count.txt

```diff
@@ -1,20 +1,76 @@
-﻿>> Count([0,""])
+﻿// ******** TABLE WITH NO BLANK RECORDS ********
+
+>> Count([0])
 1
 
->> Count([1, 2, 1/0, 4, 5])
-#Error(Kind=Div0)
+>> Count([1])
+1
 
->> Count(If(1<0,[1,2,3]))
-0
+>> Count(Table({a:1},{a:2},{a:3}))
+3
+
+>> Count([1, 2, 3, 4, 5])
+5
+
+// ******** TABLE WITH BLANK RECORDS ********
+
+>> Count([0,""])
+1
+
+>> Count([1,2,3,4,Blank()])
+4
 
 >> Count([1,"4", Blank()])
 2
 
->> Count(Table({a:1},{a:2},If(Sqrt(-1)<1,{a:3}),{a:4}))
-#Error(Kind=Numeric)
+>> Count([1,2,Blank(),Blank(),Blank()])
+2
+
+>> Count([1,2,3,If(1<0,4),Blank()])
+3
+
+>> Count(Table({a:1},{a:Blank()},{a:2},{a:Blank()} ))
+2
 
->> Count(Table({a:1},{a:2},If(1<0,{a:3}),{a:4}))
+>> Count(Table({a:1},{a:2},If(1<0,{a:3}),If(1<0,{a:4}),{a:5}))
 3
 
->> Count(Sort([-2,-1,0,1,2],1/Value))
+>> Count(LastN([1,2,3,4], Blank()))
+0
+
+>> Count([1,"1","2",Blank()])
+3
+
+//One blank record
+>> Count([Blank()])
+0
+
+//All blank records
+>> Count([Blank(),Blank(),Blank(),Blank(),Blank()])
+0
+
+>> Count(Table({a:Blank()},{a:Blank()},{a:Blank()}))
+0
+
+// ******** BLANK TABLE ********
+
+>> Count(If(1<0,[1,2,3]))
+0
+
+// ******** ERRORS ********
```

Comment:
Should we have error cases here for when the first argument is an empty string or Blank() followed by a string or number value like Count(["", 1]) and Count([Blank(), "1"])?

Reply:
Good point, will add some cases for (compiler) errors where the table doesn't have a numeric column (which is the case in your examples)